content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -6,17 +6,24 @@ individual [stages](/doc/command-reference/run) in one or more `dvc.yaml` files
 (forming a _dependency graph_, see `dvc dag`). Refer to
 [Get Started: Data Pipelines](/doc/start/data-pipelines).
 
-> Note that a helper command, `dvc stage`, is available to create and list
-> stages.
+<admon type="tip">
+
+A helper command, `dvc stage`, is available to create and list stages.
+
+</admon>
```

Comment:
Thanks for the proper admons! BTW in this case we can probably put it in line 43 and remove it from here.